kotlin-web-spring-boot-3/src/main/kotlin/com/vndevteam/kotlinwebspringboot3/application/logging/RequestFilter.kt

```diff
@@ -0,0 +1,104 @@
+package com.vndevteam.kotlinwebspringboot3.application.logging
+
+import jakarta.servlet.FilterChain
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import org.springframework.web.filter.OncePerRequestFilter
+import java.util.*
+
+class RequestFilter : OncePerRequestFilter {
+
+    companion object {
+        const val REQUEST_ID = "request-id"
```

Comment:
```suggestion
        const val REQUEST_ID = "requestId"
```
Attribute nên theo camelCase

Reply:
dùng MDC nên t xóa luôn rồi nhé